app/uk/gov/hmrc/tariffclassificationfrontend/models/Case.scala

```diff
@@ -36,5 +37,20 @@ case class Case
   keywords: Set[String] = Set.empty
 ) {
 
-  def rulingHasNotExpired: Boolean = decision.flatMap(_.effectiveEndDate).exists(_.compareTo(Instant.now) >= 0)
+  private def hasDefinedRuling: Boolean = {
+    decision match {
+      case Some(d) if d.effectiveEndDate.isDefined => true
```

Comment:
Could this match block not be `decision.flatMap(_.effectiveEndDate).isDefined`

Reply:
yes, ok